### PR TITLE
feat: generate v1 client

### PR DESCRIPTION
**Pull request by chingor13**
Towards #74

### CODE REVIEW

File: proto-google-cloud-recommender-v1/src/main/java/com/google/cloud/recommender/v1/CostProjectionOrBuilder.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2019 Google LLC
```
Comment (kolea2) [inline proto-google-cloud-recommender-v1/src/main/java/com/google/cloud/recommender/v1/CostProjectionOrBuilder.java]: 2020

Comment (kolea2) [reply proto-google-cloud-recommender-v1/src/main/java/com/google/cloud/recommender/v1/CostProjectionOrBuilder.java]: probably needs a fix in the template?

**Comment by chingor13** [comment]
Relies on the fix in: https://github.com/googleapis/synthtool/pull/383

File: proto-google-cloud-recommender-v1/src/main/java/com/google/cloud/recommender/v1/CostProjectionOrBuilder.java (hunk 1 above)
Comment (chingor13) [reply proto-google-cloud-recommender-v1/src/main/java/com/google/cloud/recommender/v1/CostProjectionOrBuilder.java]: The template was updated in synthtool

**Review by kolea2 — APPROVED** [review]

**Pull request merged**